src/schema/schema.graphql

```diff
@@ -1922,6 +1922,14 @@ input FolderActionInput {
 	zimbraId: ID
 }
 
+input TagActionRequestInput {
```

Comment:
shouldn't be this same as whatever is passed in itemAction, can we make both methods generic and have consistency?

Reply:
Done.